pipeline/cvp/Jenkinsfile-cvp.groovy

```diff
@@ -0,0 +1,108 @@
+/*
+    Pipeline script for executing CVP test suites
+*/
+// Global variables section
+
+def nodeName = "centos-7"
+def sharedLib
+def jobStatus
+
+def sendCVPUMBMessage(def ciMsg, def status) {
+    /*
+        Trigger a UMB message for successful cvp test completion
+    */
+    def msgMap = [
+        "category": "RHCEPH CVP",
+         "status": "${status}",
+        "ci": [
+            "url": "${env.JENKINS_URL}",
+            "team": "RH Ceph QE",
+            "email": "cephci@redhat.com",
+            "name": "RH CEPH"
+        ],
+        "run": [
+            "url": "${env.JENKINS_URL}/job/${env.JOB_NAME}/${env.BUILD_NUMBER}/",
+            "log": "${env.JENKINS_URL}/job/${env.JOB_NAME}/${env.BUILD_NUMBER}/console"
+        ],
+        "system": [
+            "provider": "openstack",
+            "os": "rhel"
+        ],
+        "artifact": [
+            "nvr": "${ciMsg.artifact.nvr}",
+            "scratch": "false",
+             "component": "${ciMsg.artifact.component}",
+             "type": "brew-build",
+             "id": "${ciMsg.artifact.id}",
+             "issuer": "Unknown issuer"
```

Comment:
Is "Unknown issuer" the correct value here? Seems like we should set this to something else, like "RHCEPH QE" or something?

Reply:
This content was copied from the old cvp job.
https://gitlab.cee.redhat.com/ceph/rhcs-jenkins-jobs/-/blob/master/cephci/config/definitions/rhceph-cvp-external-gating-test.yml.
I will update it to "RHCEPH QE".

Reply:
Done with the updated